terms/scalar_field.md

```diff
@@ -0,0 +1,7 @@
+# Scalar field
+
+In the context of [elliptic curve group](elliptic_curve_group.md), **scalar field** $F_r$ is the field of scalars used in the
```

Comment:
There is a problem with displaying math formulas here but I don't know how to fix it. I copied the exact same thing to try in another file but it doesn't have the same error.

Reply:
Remove hidden invalid characters.
<img width="450" alt="Screenshot 2024-03-03 at 18 12 08" src="https://github.com/sota-zk-lab/zkp-documents/assets/33181397/4366665b-9543-4b0a-8a47-5bd3b987cfa1">